clarity/contracts/multisig/multisig-ytp-yield-wbtc-59760-wbtc.clar

```diff
@@ -120,7 +120,7 @@
     (new-fee-rate-aytoken uint)
```

Comment:
As you said, I feel like not using fixed point at all in all multisig unless necessary seems much comfortable